worker/src/backgroundMigrations/README.md

```diff
@@ -0,0 +1,38 @@
+# Background Migrations
+
+Background migrations are longer running jobs that must not be complete before a new application version can
+be served correctly.
```

Comment:
Related, should we add to the requirements that these migrations generally should not be blocking and that the instance needs to be available while the migration is running? It is assumed that the instances continuously receive new traces and frontend events during the migration, which must not lead to an inconsistent state.